### PR TITLE
Beta2: Different tasks from Custom product EAV in GraphQL epic (LYNX-52)

### PR DESCRIPTION
**Pull request by loginesta**
## Purpose of this pull request

This pull request (PR) covers documentation for the following tasks:
- https://jira.corp.adobe.com/browse/LYNX-104 Rma GraphQL ReturnItemAttributeMetadata implementation of CustomAttributeMetadataInterface
- https://jira.corp.adobe.com/browse/LYNX-105 Catalog GraphQL SwatchAttributeOptionMetadata implementation of CustomAttributeOptionInterface
- https://jira.corp.adobe.com/browse/LYNX-167 Remove UID and change attribute_code and entity_type types to ID (instead of String)
- https://jira.corp.adobe.com/browse/LYNX-183 Add filters to attributesList query
- https://jira.corp.adobe.com/browse/LYNX-187 prefix, sufix, middlename and fax fields missing in setShippingAddressesOnCart and setBillingAddressOnCart graphQL mutations
- https://jira.corp.adobe.com/browse/LYNX-195 Change ProductInterface custom_attributes to custom_attributesV2

## Affected pages

- src/pages/graphql/schema/attributes/queries/attributes-form.md
- src/pages/graphql/schema/attributes/queries/attributes-list.md
- src/pages/graphql/schema/attributes/queries/custom-attribute-metadata-v2.md
- src/pages/graphql/schema/customer/queries/customer.md
- src/pages/graphql/schema/cart/mutations/set-shipping-address.md
- src/pages/graphql/schema/cart/mutations/set-billing-address.md
- src/pages/_includes/graphql/attribute-value-interface.md
- src/pages/_includes/graphql/cart-address-input-24.md
- src/pages/_includes/graphql/customer-output-24.md
- src/pages/_includes/graphql/return.md

### CODE REVIEW

**Review by jeff-matthews — COMMENTED** [review]
@keharper, after you review/approve/merge, I can tag with 2.4.7-beta notes/differentiated code blocks in #149.

**Comment by loginesta** [comment]
you can find these changes here:
https://github.com/magento-commerce/magento2ee/pull/3577

**Comment by jeff-matthews** [comment]
According to the internal Engineering wiki for the 2.4.7-beta1 release, the cutoff date for PRs for 2.4.7-beta1 was 5/12 and we've now entered the stabilization period. That indicates that this won't make it in time for 2.4.7-beta1 and we should hold it until we're ready to start merging doc PRs for 2.4.7-beta2. I expect that to be after 2.4.7-beta1 GA (6/13).

@loginesta, if any of that is incorrect, please let us know.

**Review by keharper — APPROVED** [review]
Approved, but waiting until beta1 ships to merge

File: src/pages/_includes/graphql/customer-output-24.md
Comment (svera) [inline src/pages/_includes/graphql/customer-output-24.md:7]: The link here goes to customer address attributes, it should link to customer attributes instead.

Comment (loginesta) [reply src/pages/_includes/graphql/customer-output-24.md:7]: actually the link goes nowhere since this doesn't not exist on the page which you currently are

File: src/pages/_includes/graphql/return.md
Comment (svera) [reply src/pages/_includes/graphql/return.md:52]: This do require the `deprecated` annotation indeed.

Comment (loginesta) [reply src/pages/_includes/graphql/return.md]: that was my suspicion. fixed

**Review by keharper — APPROVED** [review]

**Pull request merged**
